tutorials/synthetic_data/01-synthetic-data-electronics.ipynb

```diff
@@ -0,0 +1,2191 @@
+{
```

Comment:
<p>Is it necessary to mention "classic"?  I think DataScience 3.0 kernel is quite universal across all SageMaker notebook offering. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/python-deequ/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Neither SageMaker JupyterLab (newest SageMaker IDE) or the original SageMaker Notebooks provide DataScience 3.0 kernel. DataScience 3.0 kernel is specific to SageMaker Classic.

Given the basic Python libraries we used for data generation, I don't believe data generation experience or results will be different in different SageMaker offerings, but we wanted to document what we used. 

I can drop that line if it doesn't seem necessary.

Reply:
Sounds good. I think they might be using a universal container now.